apps/charterafrica/src/pages/api/v1/opportunities/consultation/documents.page.js

```diff
@@ -0,0 +1,13 @@
+const baseURL = "https://dc.sourceafrica.net/api";
+
+export default async function handler(req, res) {
+  const { url } = req.query;
+  try {
+    const fullURL = `${baseURL}/oembed.json?url=${url}`;
```

Comment:
Do you foresee us pulling anything else from sourceAFRICA (other than `oembed.json` that is)? If not, then `/oembed.json` should be part of `BASE_URL` since it doesn't change based on query.